apps/ai/.env.example

```diff
@@ -6,6 +6,9 @@ FRONTEND_URL=http://localhost:3000
 APP_ID=cal-ai
 APP_URL=http://localhost:3000/apps/cal-ai
 
+# This is for the onboard route. Which domain should we send emails from?
+SENDER_DOMAIN=cal.ai
```

Comment:
This needs to be added in Vercel AI App Env as well!

Reply:
added to vercel.com/cal/ai